src/custom_auth/utils/third_party_login.py

```diff
@@ -28,6 +29,20 @@ def third_party_login(serializer, request, session=False):
                     "detail": "Something went wrong :("
                 },status=500
             )
+        except ValidationError as exception:
+            logger.error(exception)
+            return Response({
+                    "status": "error",
+                    "detail": "Disallowed creation of user now."
+                }, status=400
+            )
+        except Exception as exception:
+            logger.error(exception)
+            return Response({
+                    "status": "error",
+                    "detail": "Unknown Error",
```

Comment:
It can be a future improvement that display detailed error message.
Since unknown error is so frustrating to common user. : (

Reply:
OK. 🥹
Open an issue to trace this https://github.com/NatLee/telepy/issues/23